lib/config.js

```diff
@@ -13,6 +13,14 @@ config.messages.endpoints.messages = '/messages';
 config.messages.endpoints.messagesSearch = '/messages-search';
 config.messages.endpoints.messagesBatch = '/messages-batch';
 
+config.messages.batchMessageScheduler = {};
+// Set how often the scheduler background-job will run
+config.messages.batchMessageScheduler.frequency = 'R/PT1M';
+// Set priority of this background job
+config.messages.batchMessageScheduler.priority = 0;
+// Set max amount of bedrock workers on the scheduling job
+config.messages.batchMessageScheduler.concurrency = -1;
```

Comment:
I don't think this is a config for a scheduler, it's the (partial) config for a particular job. Also, we might have more than one job in the future, so let's use this:

```js
config.messages.jobs = {};
config.messages.jobs.batchMessages = {};